ts/Core/Axis/LogarithmicAxis.ts

```diff
@@ -282,3 +287,13 @@ interface LogarithmicAxis extends Axis {
 LogarithmicAxis.compose(Axis); // @todo move to factory functions
 
 export default LogarithmicAxis;
+
+/**
+ * Activates rendering of negative logarithmic values.
+ *
+ * @type      {boolean}
+ * @since     8.1.0
+ * @apioption xAxis.allowNegativeLog
```

Comment:
Shouldn't this option be related only to the yAxis (not xAxis)?

Reply:
Every axis could be logarithmic: http://jsfiddle.net/wgy5qnf9/
You just have to use point objects, because the array index is not sufficient.

Reply:
Ahh, you are right, my mistake!